cmd/rqlite/main.go

```diff
@@ -6,6 +6,7 @@ import (
 	"crypto/x509"
 	"encoding/json"
 	"fmt"
+	httpcl "github.com/rqlite/rqlite/http"
```

Comment:
I prefer non-standard-library imports to be group separately, and in a second block, is the style in this file already.